3p/iframe-messaging-client.js

```diff
@@ -57,6 +57,15 @@ export class IframeMessagingClient {
   }
 
   /**
+   *  Send a postMessage to Host Window
+   *  @param {object} message The message to send.
+   *  @private
```

Comment:
ah, sorry. it ran fine in my examples/tests but apparently babel doesn't enforce the same rules. made a new pr to fix it